monitoring/prometheus/alerts/ceph_default_alerts.yml

```diff
@@ -27,22 +27,86 @@ groups:
+          severity: warning
+          type: ceph_default
+        annotations:
+          documentation: https://docs.ceph.com/en/latest/rados/operations/health-checks/#mon-down
+          description: |
+            {{ $down := query "count(ceph_mon_quorum_status == 0)" | first | value }}{{ $s := "" }}{{ if gt $down 1.0 }}{{ $s = "s" }}{{ end }}You have {{ $down }} monitor{{ $s }} down.
+            Quorum is still intact, but the loss of further monitors will make your cluster inoperable.
 
             The following monitors are down:
             {{- range query "(ceph_mon_quorum_status == 0) + on(ceph_daemon) group_left(hostname) (ceph_mon_metadata * 0)" }}
               - {{ .Labels.ceph_daemon }} on {{ .Labels.hostname }}
             {{- end }}
+      - alert: Ceph mon disk space critically low
+        expr: ceph_health_detail{name="MON_DISK_CRIT"} == 1
+        for: 1m
+        labels:
+          severity: critical
+          type: ceph_default
+        annotations:
+          documentation: https://docs.ceph.com/en/latest/rados/operations/health-checks/#mon-disk-crit
+          description: |
+            The free space available to a monitor's store is critically low (<5% by default).
+            You should increase the space available to the monitor(s). The
+            default location for the store sits under /var/lib/ceph. Your monitor hosts are;
+            {{- range query "ceph_mon_metadata"}}
+              - {{ .Labels.hostname }}
+            {{- end }}
+
+      - alert: Ceph mon disk space running low
+        expr: ceph_health_detail{name="MON_DISK_LOW"} == 1
+        for: 5m
+        labels:
+          severity: warning
+          type: ceph_default
+        annotations:
+          documentation: https://docs.ceph.com/en/latest/rados/operations/health-checks/#mon-disk-low
+          description: |
+            The space available to a monitor's store is approaching full (>70% is the default).
+            You should increase the space available to the monitor store. The
+            default location for the store sits under /var/lib/ceph. Your monitor hosts are;
+            {{- range query "ceph_mon_metadata"}}
+              - {{ .Labels.hostname }}
+            {{- end }}
+
+      - alert: Clock skew detected across Ceph Monitor daemons
+        expr: ceph_health_detail{name="MON_CLOCK_SKEW"} == 1
+        for: 1m
+        labels:
+          severity: warning
+          type: ceph_default
+        annotations:
+          documentation: https://docs.ceph.com/en/latest/rados/operations/health-checks/#mon-clock-skew
+          description: |
+            The ceph monitors rely on a consistent time reference to maintain
+            quorum and cluster consistency. This event indicates that at least
+            one of your mons is not sync'd correctly.
+
+            Review the cluster status with ceph -s. This will show which monitors
+            are affected. Check the time sync status on each monitor host.
```

Comment:
How do we make sure that 

1. the url gets updated if the underlying documentation moves?
2. Those two copies of the docs stay in sync?  

This looks like a maintenance nightmare to me.

Reply:
Yeah, maintaining cross-component links healthy is a tough duty... However the value of connecting separated information and creating a unified context is IMHO the best advantage of UIs compared CLI/plain-text interfaces. So IMHO it's worth the effort.

How can we fix this:
* We have an issue with `latest` or `master` in Docs URLs as that doesn't allow us to provide permalinks (`/latest` will eventually become `/quincy` and so on) . We should create permalink URLs (including the commit ID) in the Read the Docs platform. @tchaikov @zdover23 how can we do this? 
* We should be able to connect the alerts to URLs or URLs#fragments, and ensure those remain stable on a PR basis.
* Dashboard already has a tox script (invoked in `make check`) that cross-checks Grafana UUIDs and the Dashboard UI HTML code. Something similar could be done to ensure that the linked doc URLs exist. Sphinx provides a [way to generate an inventory](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#finding-the-reference-name) of references that can be used for validation.